cmake/windows-setup.cmake

```diff
@@ -7,7 +7,11 @@ if(FLB_WINDOWS_DEFAULTS)
   message(STATUS "Overriding setttings with windows-setup.cmake")
   set(FLB_REGEX                 Yes)
   set(FLB_BACKTRACE              No)
-  set(FLB_LUAJIT                Yes)
+  if (CMAKE_SYSTEM_PROCESSOR MATCHES "^(ARM64|AARCH64)")
```

Comment:
Probably should link out to why if there is a good URL for an issue, etc. so we can enable if/when LUA is supported. We probably need to document this as well, presumably this means no LUA filter or does it just reduce functionality in another way?

Reply:
There is a meta issue for supporting LuaJIT on ARM64 Windows: https://github.com/fluent/fluent-bit/pull/6621/commits/186ec58892ad8fc80b6a47f87fccb9e2fee1111d
For ARM64 Windows, we should disable for now and will revisit to enable this functionality.